idunn/utils/prometheus.py

```diff
@@ -0,0 +1,47 @@
+import time
+import contextlib
+
+from apistar import Component
+from prometheus_client import Counter, Gauge, Histogram
+
+
+""" The logic of the Prometheus metrics is defined in this module """
+
+
+IDUNN_WIKI_REQUEST_DURATION = Histogram(
+    "idunn_wiki_request_duration_seconds",
+    "Time spent processing a Wiki request.",
+    ["target", "handler"],
+)
+
+IDUNN_WIKI_EXCEPTIONS_COUNT = Counter(
+    "idunn_exceptions_count",
+    "Number of exceptions caught in Idunn.",
+    ["exception_type"]
+)
+
+IDUNN_WIKI_BREAKER_ERRORS_COUNT = Counter(
```

Comment:
I'm not sure we want custom counter for the breaker and rate_limiter

Reply:
done